README.md

```diff
@@ -18,21 +18,63 @@ The plugin requires the cypress binary. https://www.npmjs.com/package/cypress
 
 The plugin is able to interact with the django server and database. Therefore it is important to provide the djangopath when executing the tests. The djangopath should be the root directory of your project. The plugin assumes to have a `manage.py` to interact with the django server.
 
-### database scripts
+### database
 
-The following database script need to be provided so the plugin can properly interact with the database:
+The following database script need to be provided in the django-root
```

Comment:
Besser: The following database scripts need to be provided in the project root

Reply:
🌷